### ISSUE TITLE
Uncaught TypeError: Cannot read property 'nodeType' of null  at editOnInput

### ISSUE THREAD

**Issue by pietrop** [facebook/draft-js#2204]
**Do you want to request a *feature* or report a *bug*?**

A bug 

**What is the current behavior?**

DraftJs is used to power a timed transcript editor. ([@bbc/react-transcript-editor](https://github.com/bbc/react-transcript-editor)) with  `blockRendererFn`  used to have a `WrapperBlock` to display speaker names and time-codes at paragraph level.

Trying to add a html `select` element to make it easier to change speaker labels within a `WrapperBlock` in this PR. https://github.com/bbc/react-transcript-editor/pull/210

However it seems that whenever I make a selection with the select element I get this error in the coneole.

```
Uncaught TypeError: Cannot read property 'nodeType' of null
    at editOnInput
```

![Screenshot 2019-10-09 at 12 38 29](https://user-images.githubusercontent.com/4661975/66480204-fc928c00-ea95-11e9-969b-8b269228ee4d.png)

I am not quiet sure what this means or it might refer to?
I found this helpful issue/comment https://github.com/facebook/draft-js/issues/1085 but would love to learn more about the inner workings to understand why, if my hypothesis is right, calling select in a `WrapperBlock` might be triggering this?

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem. You can use this jsfiddle to get started: https://jsfiddle.net/gmertk/e61z7nfa/.**

To reproduce checkout this PR https://github.com/bbc/react-transcript-editor/pull/210

1. git clone repo / branch
```
git clone -b speaker-dropdown-select-2 git@github.com:pietrop/react-transcript-editor.git
```
2. cd into folder - `cd react-transcript-editor`
3. `npm install`
4. `npm start`
5. storybook starts on [http://localhost:6006](http://localhost:6006)
6. click button `load demo`
7. click on one of the speaker names, and select a new one
8. watch error in console 

**What is the expected behavior?**

To not get a console error message when using select in the WrapperBlock

**Which versions of Draft.js, and which browser / OS are affected by this issue? Did this work in previous versions of Draft.js?**

initially noticed it in `0.10.5`, then updated to `0.11.1` but the message persists.

**Comment by gland2015** [comment]
wrap block children with a <div>，Especially when block'text.length is ''

**Comment by gland2015** [comment]
wrap block children with a div，Especially when block'text.length is ''

**Comment by skyitachi** [comment]
anchorNode can be null

**Comment by mrkev** [comment]
Merged #2330  to fix this.
